api/communication-api/src/main/java/com/cognifide/aet/communication/api/execution/ProcessingStatus.java

```diff
@@ -35,6 +35,11 @@
    */
   FATAL_ERROR,
 
+  /**
+   * An error occured when accessing database.
+   */
+  PROCESSING_ERROR,
```

Comment:
If this status is only returned when the database does not exist (and can't be created), please rename it to e.g. `MISSING_DATABASE`